scripts/ci/ensure-angular-dependencies-compatibility.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
> https://travis-ci.org/resin-io/etcher/jobs/209813569 fails with the same error, but TravisCI thinks the job passed! Eeek, do we need to tweak something somewhere so that if npm run sanity-checks fails then the whole CI job fails?

Oh, interesting. I wonder why we didn't catch this before. Looks like the commands inside conditionals should be chained with `&&` rather than terminated with a `;`.